src/utilities/util_verify.c

```diff
@@ -128,12 +145,16 @@ util_verify(WT_SESSION *session, int argc, char *argv[])
           "-u (unredact all data), should not be set to true simultaneously with -k (unredact only "
           "keys)");
 
+    if (dump_offsets != NULL && ckpt != NULL)
```

Comment:
Given your other PR https://github.com/wiredtiger/wiredtiger/pull/11950, should I wait for that one to be merged before reviewing further here?

Reply:
I am not sure about this. I think we can merge them independently and when one is merged, I'll just update the other one.